.editorconfig

```diff
@@ -0,0 +1,31 @@
+[*]
```

Comment:
Is this a config file? Do we want this file as part of merge or ?

Reply:
Yes I added this to move a bit in the direction of having a standard way of configuring the editor. This file is (in theory) globally supported by most editors and we can set the indent, spacing and tabs in here for everything. So I think it's a good idea to commit it to force code styling in future. 

For now it has basic configurations but we can extend it more in the future.